sigs.yml

```
@@ -0,0 +1,502 @@
+- name: Specification SIGs
+  sigs:
+  - name: 'Specification: General'
+    meeting: Every Tuesday at 08:00 PT
+    notes:
+      type: gDoc
+      value: 1pdvPeKjA8v8w_fGKAN68JjWBmVJtPCpqdi9IZrd6eEo
+    chat:
+      - type: slack
+        name: '#otel-specification'
+        id: C01N7PP1THC
+    invites: calendar-spec-general
+    tcSponsors: []
+    gcLiaison: []
+  - name: 'Specification: Logs'
+    meeting: 'Merged into "Specification: General"'
+    notes:
+      type: none
+      value: See above
+    chat:
+      - type: slack
+        name: '#otel-logs'
+        id: C01N5UCHTEH
+    invites: none
+    tcSponsors:
+    - name: Tigran Najaryan
+      github: tigrannajaryan
+    gcLiaison: []
+  - name: 'Specification: Sampling'
+    meeting: Every Thursday at 08:00 PT
+    notes:
+      type: gDoc
+      value: 1gASMhmxNt9qCa8czEMheGlUW2xpORiYoD7dBD7aNtbQ
+    chat:
+      - type: slack
+        name: '#otel-sampling'
+        id: C027DS6GZD3
+    invites: calendar-spec-sampling
+    tcSponsors:
+    - name: Josh MacDonald
+      github: jmacd
+    gcLiaison: []
```

```suggestion
    gcLiaison: []
```
@open-telemetry/governance-committee please assign yourself for being the liaison person for this SIG

---

I volunteer